src/components/drops/drop/useMakeUpdatePosition.js

```diff
@@ -49,6 +49,8 @@ const getYPosition = (align, targetRect, dropRect, canHideTarget = true) => {
 
   const minY = Math.max(0, y)
 
+  y = Math.min(window.innerHeight - dropRect.height, minY)
```

Comment:
Which case was it trying to solve in the first place? Getting out of screen.. where?

Reply:
It was trying to solve the following:
When the screen doesn't have enough height, the dropdown menus of header items render outside of the screen and not at position `y` of the element they are connected to

What was caused by this fix:
Apart from `Drop`, `Menu` also uses the hook, and the menu rendered for user profile, settings, etc. is now misplaced, outside the screen at its bottom

Reply:
I guess you are referring to the issue I was trying to solve in first place, but it does not!
I re-checked it now! 😄 
Since my fix was trying to solve an issue happening when `console` is open, and for all this time there was no issue reported by any of our users, I guess it is better to revert it and open a ticket to add an extra prop to `Drop` component if we need to skip this recalculation!

Reply:
yeah, also we dont need to lose time on fixing the app working on really narrow screens 😄

Reply:
Ok! So, I am leaving it as it is!

Reply:
By that we mean that we keep the previous impl.
This `y = Math.min(window.innerHeight - dropRect.height, minY)` should remain from what I got from our convey. (sorry about writing convey, I was bored to write conversation, but now I wrote more)